### PR TITLE
remove Add Machine button

### PR DESCRIPTION
**Pull request by dantrainor**
Based on the feature freeze, we'll remove the Add Machine button through the frontend
and revisit this after beta.

### CODE REVIEW

File: frontend/packages/metal3-plugin/src/components/machine-cell.tsx [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
   host: K8sResourceKind;
-  status: HostMultiStatus;
+  HostMultiStatus;
 }
 
-const MachineCell: React.FC<MachineCellProps> = ({ host, status }) => {
+const MachineCell: React.FC<MachineCellProps> = ({ host }) => {
   const machineName = getHostMachineName(host);
 
   const {
```
Comment (jtomasek) [inline frontend/packages/metal3-plugin/src/components/machine-cell.tsx]: Please replace this metadata definition with:
```suggestion
  const namespace = getNamespace(host)
```

File: frontend/packages/metal3-plugin/src/components/machine-cell.tsx [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
+import { ResourceLink } from '@console/internal/components/utils';
 import { referenceForModel, K8sResourceKind } from '@console/internal/module/k8s';
 
 import { getHostMachineName } from '../selectors';
-import { canHostAddMachine, HostMultiStatus } from '../utils/host-status';
 
 interface MachineCellProps {
   host: K8sResourceKind;
-  status: HostMultiStatus;
+  HostMultiStatus;
```
Comment (jtomasek) [inline frontend/packages/metal3-plugin/src/components/machine-cell.tsx]: Remove this, host props are now just `host`

File: frontend/packages/metal3-plugin/src/components/host.tsx [hunk 3]
```diff
@@ -99,9 +98,7 @@ const HostsTableRow: React.FC<HostsTableRowProps> = ({
       <TableData className={tableColumnClasses[1]}>
         <BaremetalHostStatus status={status} />
       </TableData>
-      <TableData className={tableColumnClasses[2]}>
-        <NodeCell nodeName={nodeName} namespace={namespace} />
-      </TableData>
+      <TableData className={tableColumnClasses[2]} />
```
Comment (jtomasek) [inline frontend/packages/metal3-plugin/src/components/host.tsx]: This is incorrect probably due to the rebase. This should not be changed.

File: frontend/packages/metal3-plugin/src/components/host-detail.tsx [hunk 4]
```diff
@@ -111,14 +108,6 @@ const BaremetalHostDetails: React.FC<BaremetalHostDetailsProps> = ({
               <br />
               NICs: {ips}
             </dd>
-            {(canHostAddMachine(status.status) || machineName) && (
```
Comment (jtomasek) [inline frontend/packages/metal3-plugin/src/components/host-detail.tsx]: This should not be removed, just check only machineName presence:
```
{machineName && (
...
```

File: frontend/packages/metal3-plugin/src/components/host.tsx (hunk 3 above)
Comment (dantrainor) [reply frontend/packages/metal3-plugin/src/components/host.tsx]: It's what linter suggested.  I did this to preserve what I understood was the order in which the cells of the table were rendered.  What would be a better way to do this?  Modify the index used to reflect one of the datas no longer being present (e.g. decrement the number by 1 for data that follows this removal)

Comment (jtomasek) [reply frontend/packages/metal3-plugin/src/components/host.tsx]: Originally this `<TableData>` rendered a `MachineCell`, now it correctly renders `NodeCell`. This PR should not do any changes to this part of the code.

**Comment by dantrainor** [comment]
This has since (recently) been included in work that @jtomasek had done but not yet created a PR for.  I'll close this one out, and use @jtomasek's PR to update the task with and track it that way.

**Pull request closed without merging**